README.md

```diff
@@ -39,12 +39,10 @@ dependency of Argo, so you don't need to specify it in your Cartfile.
 
 [CocoaPods]: http://cocoapods.org
 
-__DISCLAIMER: CocoaPods doesn't officially support Swift projects yet. Use the
```

Comment:
I think we should keep the disclaimer and change it to make it very, very clear that you need version 0.36.0 of Cocoapods otherwise you're going to have a bad time. Then we can remove the line below about installing with that particular version.